.github/workflows/cron_nightly_build.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow aim to create nightly builds on active branches
+name: Nightly Build
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  nightly-build:
+    name: Nightly Build
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        BRANCH:
+          - develop
+          - 1.7.8.x
+
+    env:
+      RELEASE_DIR: '/tmp/ps-release'
+      GH_BRANCH: ${{ matrix.BRANCH }}
+      GC_PROJECT_ID: ${{ secrets.GC_PROJECT_ID }}
+      GC_SERVICE_KEY: ${{ secrets.GC_SERVICE_KEY }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ env.GH_BRANCH }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
+
+      - name: Create Release directory
+        run: mkdir -p ${{ env.RELEASE_DIR }}
+
+      - name: Create build
+        run: php tools/build/CreateRelease.php --destination-dir=${{ env.RELEASE_DIR }}
```

Comment:
You need nodejs 14 at least in order to be able to create release

Reply:
done